src/OpenTelemetry/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Removes upper constraint for Microsoft.Extensions.Logging
   dependencies. ([#2179](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2179))
 
+* Fix bug which caused ILogger.Log calls to throw exception, when the
```

Comment:
Do we consider this as a bug?

Reply:
Good qn. After seeing some other implementations, which throw in Log(...formatter=null)., I am thinking we should follow the regular pattern of "not throwing from instrumentation APIs after successful initialization". So Otel logging provider can chose to not throw, end user may still see exception from other providers they have enabled.

And I can reword the changelog to just say:
"OpenTelemetry Logger modified to not throw when formatter is null"